### ISSUE TITLE
Autocomplete's aria-live region should clear itself out so it can't be navigated to

### ISSUE THREAD

**Issue by WestonThayer** [adamsilver/nostyle#18]
This happens in any screen reader. For example, with NVDA + Firefox:

1. Go to https://nostyle.herokuapp.com/examples/autocomplete
2. Tab to the control to focus it and type "states"
3. Use `NVDA + SPACE` to switch out of forms mode and into browse mode
4. Press `ARROW_DOWN`. Virtual cursor will be moved to the little dropdown arrow, which announces as "clickable" (could be an issue in and of itself)
5. Press `ARROW_DOWN` again

Virtual cursor is moved to the hidden aria-live region, where user hears "3 results".

![screenshot of NVDA focus highlight](https://user-images.githubusercontent.com/709153/80653733-a5eba000-8a2f-11ea-984a-f7c1e6882781.png)

This can be fixed by clearing out the contents of the aria-live region 500ms after it's been added.

**Comment by WestonThayer** [comment]
Also, sorry if you're not looking for issues yet! Your awesome library was linked to in the web-a11y Slack. I was totally impressed by autocomplete!

**Comment by adamsilver** [comment]
Thanks a lot for this @WestonThayer
